src/main/java/org/kainos/ea/resources/AuthController.java

```diff
@@ -59,6 +56,24 @@ public Response login(Login login) {
         }
     }
 
+    @POST
+    @Path("/register")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Registers a user")
+    public Response register(RegisterRequest request) {
+        try {
+            authService.register(request);
+
+            return Response.status(Response.Status.CREATED).build();
+        } catch (FailedToRegisterException e) {
```

Comment:
This exception could be thrown for any issue from the DB, it doesn't mean that it was down to user input, so 400 doesn't feel right here